src/environments/environment.model.ts

```diff
@@ -111,4 +116,26 @@ export const ENVIRONMENT_DEFAULTS: Environment = {
     { lang: 'de_DE', currency: 'EUR', value: 'de', displayName: 'German', displayLong: 'German (Germany)' },
     { lang: 'fr_FR', currency: 'EUR', value: 'fr', displayName: 'French', displayLong: 'French (France)' },
   ],
+  cookieConsentOptions: {
+    options: [
+      {
+        id: 'required',
+        name: 'cookie.consent.option.required.name',
+        description: 'cookie.consent.option.required.description',
+        required: true,
+      },
+      {
+        id: 'functional',
+        name: 'cookie.consent.option.functional.name',
+        description: 'cookie.consent.option.functional.description',
+      },
+      {
+        id: 'tracking',
+        name: 'cookie.consent.option.tracking.name',
+        description: 'cookie.consent.option.tracking.description',
+      },
+    ],
```

Comment:
again, I prefer the idea of having a map instead of an array. But this is your choice in the end 😃

Reply:
switched from array to map.